tests/test_browser.py

```diff
@@ -2744,26 +2716,18 @@ def test_sdl2_key(self):
         }
 
         function keydown(c) {
-          var event = document.createEvent("KeyboardEvent");
-          event.initKeyEvent("keydown", true, true, window,
-                             0, 0, 0, 0,
-                             c, c);
+          var event = new KeyboardEvent("keydown", { 'keyCode': c, 'charCode': c, 'view': window, 'bubbles': true, 'cancelable': true });
           var prevented = !document.dispatchEvent(event);
 
           //send keypress if not prevented
           if (!prevented) {
-            event = document.createEvent("KeyboardEvent");
-            event.initKeyEvent("keypress", true, true, window,
-                               0, 0, 0, 0, 0, c);
+            var event = new KeyboardEvent("keypress", { 'keyCode': c, 'charCode': c, 'view': window, 'bubbles': true, 'cancelable': true });
             document.dispatchEvent(event);
           }
         }
 
         function keyup(c) {
-          var event = document.createEvent("KeyboardEvent");
-          event.initKeyEvent("keyup", true, true, window,
-                             0, 0, 0, 0,
-                             c, c);
+          var event = new KeyboardEvent("keyup", { 'keyCode': c, 'charCode': c, 'view': window, 'bubbles': true, 'cancelable': true });
```

Comment:
Seems like this is repeated a lot.

Could we use a helper so we can just write e.g. `makeKeyEvent(c)`?

Reply:
sgtm